### DISCUSSION TITLE
Объясните что это за программа

### DISCUSSION THREAD

**Discussion by EgoBrainProgrammer** [hufrea/byedpi#142] [Q&A]
Здравствуйте. Прошу мне объяснить на пальцах, что делает эта программа.
У меня есть некоторые догадки, например, что это некий сетевой шлюз или прокси сервер SOCKS 5.
Прошу "разжевать" подробно, что она делает и как мне через компьютер, на котором она запущена пустить трафик.
Схема у меня такая Router 2 -> Rasberry Pi (ByeDPI) -> Router 1 -> WAN.
К Router 2 я собираюсь подключить телевизоры и телефоны, чтобы был доступен YouTube.

**Comment by jura1243** [comment]
запускаете программу с параметрами. в браузере пишете чтоб инет брал из прокси сокс с адресом например 127.0.0.1:1080 . смотрите ютюб как раньше.

**Comment by EgoBrainProgrammer** [comment]
Нашёл ответ, здесь разжевано
https://habr.com/ru/articles/584162/

Ну а сама программа (спасибо за неё автору) является SOCKS 5 прокси сервером, о чем в readme и написано :)
